woocommerce/woocommerce-android
# Issue/6585 add dismiss button to banner

Part of: #6585 


### Description
This PR adds a dismiss `X` button to the install WC Shipping banner from order detail screen. 

### Testing instructions

Using a site eligible for shipping labels, disable the WC Shipping plugin  from `wc-core` and test that the banner is shown if this conditions are met: 

- The store is located in US.
- The order currency is USD.
- The products are eligible for SL (not virtual/downloadable).
- The order is not eligible for IPP.

Click on dismiss button and check that the banner is hidden and never shown again for that site

### Images/gif


https://user-images.githubusercontent.com/2663464/171105002-0b5351d6-821d-4e3e-859f-2276a104d2a7.mp4
